src/coreComponents/physicsSolvers/wavePropagation/AcousticWaveEquationSEM.hpp

```diff
@@ -31,7 +31,7 @@ class AcousticWaveEquationSEM : public WaveSolverBase
 {
 public:
 
-  using EXEC_POLICY = serialPolicy;//parallelDevicePolicy< 32 >;
+  using EXEC_POLICY = parallelDevicePolicy< 32 >;
```

Comment:
@rrsettgast Hello, before we had parallelDevicePolicy without the 32, what is the best solution ?